src/Card/Card.Headline.js

```diff
@@ -0,0 +1,79 @@
+import PropTypes from "prop-types";
+import { dcnb } from "cnbuilder";
+import React from "react";
+import { Heading } from "../Heading/Heading";
+import {
+  headingFonts,
+  headingLevels,
+  headingSizes,
+  headingWeights,
+} from "../Heading/Heading.levers";
+import intKeys from "../common/util/intKeys";
+
+/**
+ * Headline component
+ * @param {*} param0
+ * @returns
+ */
+
+export const Headline = ({
+  children,
+  level,
+  size,
+  weight,
+  font,
```

Comment:
Probably want to set defaults for `weight` and `font` as well unless you don't expect the developer to always use these.

Reply:
never mind. I see the default props.